tests/sources/json/test_aardvark.py

```diff
@@ -147,14 +147,54 @@ def test_aardvark_record_is_deleted_returns_false_if_value_is_false(
     assert MITAardvark.record_is_deleted(source_record) is False
 
 
-def test_aardvark_get_alternate_titles_success(aardvark_record_all_fields):
-    assert MITAardvark.get_alternate_titles(next(aardvark_record_all_fields)) == [
+def test_aardvark_get_alternate_titles_success(aardvark_records):
+    source_record = next(aardvark_records)
```

Comment:
Stub records are a lot different with JSON vs. XML so I don't fully approve of this particular approach but figure we would discuss in this PR before I go further. All suggestions welcome so we can properly set our JSON source testing conventions going forward!

Reply:
I'd agree that JSON records are easier to manipulate, given that once was parsed it's just a dictionary we can easily modify.

Instead of using the overhead of an iterator and `next(...)`, what about a similar stub method -- for symmetry -- that just returns a barebones dictionary?  Then the test is free to add/remove keys as needed.

While I think a test could just do:
```python
source_record = {}
```

The benefit of a stub might be that we could set some shared defaults, maybe bits and bobs that we know are commonly accessed.  But at the end of the day, it _is_ just a dictionary.

Reply:
You are correct about the value of symmetry, I'll create a method!